### ISSUE TITLE
Conditionally create things in MySolarSystemScreenView that are specific to the Lab screen.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/my-solar-system#198]
For PhET-iO instrumentation...  In MySolarSystemScreenView, LabModeComboBox is always instantiated, but is invisible in the Intro screen.  This will need to change for PhET-iO.  LabModeComboBox (and its enclosing panel) should be conditionally instantiated.

```ts
    const labModeComboBox = new LabModeComboBox( model, this.topLayer, {
      widthSizable: false,
      layoutOptions: {
        align: 'center'
      }
    } );
...
        new Panel( new Node( { children: [ labModeComboBox ], visible: model.isLab } ), SolarSystemCommonConstants.CONTROL_PANEL_OPTIONS ),
```

**Comment by pixelzoom** [comment]
There are a few other things that are specific to the Lab screen that need to be conditionally created. Search for `visible: model.isLab`.

**Comment by pixelzoom** [comment]
We should also consider moving these out of MySolarSystemScreenView and into LabScreenView.  That would be the better OO approach.

**Comment by pixelzoom** [comment]
Note that in https://github.com/phetsims/my-solar-system/issues/205, labModePanel was moved out of MySolarSystemScreenView, and into LabScreenView.  Similarly labModeProperty was moved out of SolarSystemCommonModel, and into LabModel.

**Comment by pixelzoom** [comment]
We've found that conditionally creating things, or moving them to subclasses, is sometimes too difficult/costly at this point.  So instead we've resorted to flags that tell us whether to instrument or opt-out, like these examples in MySolarSystemScreenView:

```ts
    const moreDataCheckbox = MySolarSystemCheckbox.createMoreDataCheckbox( this.visibleProperties.moreDataVisibleProperty,
      model.isLab ? valuesPanelTandem.createTandem( 'moreDataCheckbox' ) : Tandem.OPT_OUT );

    const unitsInformationDialog = new UnitsInformationDialog(
      model.isLab ? valuesPanelTandem.createTandem( 'unitsInformationDialog' ) : Tandem.OPT_OUT );
```

I can't say that I love (or even like) this pattern.  But for retrofitting PhET-iO to an existing implementation, it's a useful tool.  We should be careful not to abuse it by using it too much, or in situations where doing better is not unacceptably difficult/costly.

That said... Instrumentation of SolarSystemCommonScreenView is complete.  So closing this issue.
